ledger/accountdb.go

```diff
@@ -179,11 +213,51 @@ func writeCatchpointStagingBalances(ctx context.Context, tx *sql.Tx, bals []enco
 	return nil
 }
 
+// createCatchpointStagingHashesIndex creates an index on catchpointpendinghashes to allow faster scanning according to the hash order
+func createCatchpointStagingHashesIndex(ctx context.Context, tx *sql.Tx) (err error) {
+	_, err = tx.ExecContext(ctx, "CREATE INDEX IF NOT EXISTS catchpointpendinghashesidx ON catchpointpendinghashes(data)")
```

Comment:
That would be faster if we were to insert the entries into that table ordered; but since the items we have aren't ordered, inserting them to the table results in many reallocations which makes it slower.
( i.e. slower compared to adding a index once the table is ready, naturally )

Reply:
btw - be aware that when creating a table without primary key, the "real" primary key is the "rowid". any created index index into that rowid. I don't know if the underlying implementation of a blob primary key doesn't end up with an index that points into the rowid.